test/test-apppath.js

```diff
@@ -7,34 +7,34 @@ var assert  = yeoman.assert;
 var fs      = require('fs');
 var test    = require('./helper.js');
 
+var config = [
+  '{',
+  '  "generator-backbone": {',
+  '    "appPath": "public",',
+  '    "appName": "Temp"',
+  '  }',
+  '}'
+].join('\n');
```

Comment:
Yes :blush: the only reason I left it like that was to keep the same structure used before (I tried to keep modifications confined to only those required by the yeoman API changes)

I can change this as part of a cleanup PR in the future **or** update all the appearances of this weird json constructions now, it's up to you to decide :wink: